Application Lifecycle Management/instances-api.md

```diff
@@ -333,85 +390,170 @@ Creates a new instance and gets the created instance.
+                "factor": 100000
+            },
+            "image": "Linux Compute",
+            "instances": 1,
+            "keypair": "None",
+            "cloud": "EC2",
+            "volumes": [],
+            "flavor": "m1.small",
+            "schema": "http://elasticbox.net/schemas/aws/ec2/profile",
+            "managed_os": false,
+            "location": "us-east-1"
+        },
+        "provider_id": "0669730b-6f01-421b-8a5e-597bca20c1bf",
+        "automatic_updates": "off",
+        "name": "default-small-us-east-1",
+        "created": "2018-12-11 13:53:56.512649",
+        "deleted": null,
+        "variables": [],
+        "updated": "2018-12-11 13:53:56.512649",
+        "visibility": "workspace",
+        "readme": {
+            "url": "services/resources/default_box_overview.md",
+            "upload_date": "2018-12-11 13:53:56.512131",
+            "length": 1307,
+            "content_type": "text/x-markdown"
+        },
+        "members": [],
+        "claims": [
+            "small",
+            "linux"
+        ],
+        "owner": "operations",
+        "organization": "centurylink",
+        "id": "11c8882e-3888-4bdd-bd2d-b92766a423c7",
+        "schema": "http://elasticbox.net/schemas/boxes/policy"
+    },
+    "uri": "/services/instances/i-b1qtkz"
 }
 ```
 
 ### POST /services/instances
 Register an unregistered instance: notice that it's the same endpoint to create a new instance.
 
-**Normal Response Codes**
-
-* 202
-
-**Error Response Codes**
+### URL
 
-* Invalid Data (400)
-
-**Request Parameters**
-
-| Parameter | Type |Description |
-|-----------|------|------------|
-| schema | string | Required. Register Instance Request schema URI. |
-| owner | string | ID of the workspace where the instance is imported. |
-| name | string | Instance name to register. |
-| description | string | Instance description to register. |
-| unregistered_id | string | Instance unique identifier |
-| linux_username | string | Linux user to install agent |
-| private_key | string | Linux key to install agent  |
-| windows_username | string | Windows username to install agent |
-| windows_password | string | Windows password to install agent |
-| instance_tags | array | List of tags defined at deployment time. |
-| automatic_updates | string | One of the: major, minor, patch, off. Default off. |
-| lease | array | Schedules an instance with two parameters:<li>expire. Specifies in UTC format YYYY-MM-DD HH:MM:SS.SSSSSS, the time and date for stopping an instance. It’s required only when an instance is set to terminate or shut down.</li><li>operation. Specifies an instance to stop with **shutdown** or **terminate**. When not scheduled, the instance is set to **alwayson**.</li> |
+#### Structure
 
 ```
-Headers:
+[POST] /services/instances
+```
 
-Content-Type: application/json
-Authorization: Bearer your_json_web_token
-ElasticBox-Release: 4.0
+#### Example
+```
+[POST] https://cam.ctl.io/services/instances
 ```
 
```

Comment:
Place this block (lines 556-606) after Request - Uri Parameters table of this command